src/schema/firefox-schemas-import.js

```diff
@@ -8,6 +8,12 @@ import request from 'request';
 import tar from 'tar';
 
 const FLAG_PATTERN_REGEX = /^\(\?[im]*\)(.*)/;
+export const FLAG_PATTERN_REWRITES = {
+  '(?i)^\\{[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\\}$':
+    // eslint-disable-next-line max-len
```

Comment:
Can we maybe add that to lines 11-16 to avoid having the comment in the midst of the regex?